app/_src/networking/transparent-proxying.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Transparent Proxying
+---
+
+## What is Transparent Proxying?
+A transparent proxy is a type of server that can intercept network traffic to and from a service without changes to the client application code. In the case of {{site.mesh_product_name}} it is used to capture traffic and redirect it to `kuma-dp` so Mesh policies can be applied.
+
+To accomplish this, {{site.mesh_product_name}} utilizes [`iptables`](https://linux.die.net/man/8/iptables).
+
+Below is a high level visualization of how Transparent Proxying works
+
+{% mermaid %}
+ sequenceDiagram
+ autonumber
+     participant Browser as Client<br>(e.g. mobile app)
+     participant Firewall as Firewall
```

Comment:
I think "firewall" isn't the right terminology here, maybe just "kernel" or "operating system".